docs/config/README.md

```diff
@@ -1631,23 +1631,34 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable      | Default         | Description                           |
+| Option        | Default         | Description                           |
 | ------------- | --------------- | ------------------------------------- |
-| `style_root`  | `"bold red"`    | The style used when the user is root. |
-| `style_user`  | `"bold yellow"` | The style used for non-root users.    |
+| `format`            | `"via [$user]($style) "` | The format for the module.                                                       |
+| `style`  | (variable based on user or root)    | The style for the module. |
 | `show_always` | `false`         | Always shows the `username` module.   |
 | `disabled`    | `false`         | Disables the `username` module.       |
 
+### Variables
+
+| Variable     | Example         | Description                                                         |
+| ------------ | --------------- | ------------------------------------------------------------------- |
+| `style_root` | `"red bold"`    | Mirrors the value of option `style` when root is logged in.         |
+| `style_user` | `"yellow bold"` | Mirrors the value of option `style` when regular user is logged in. |
```

Comment:
I think the option here should be `style` e.g

```suggestion
| `style` | `"red bold"`    | Mirrors the value of option `style_root` when root is logged in and `style_user` otherwise. |
```